packages/ra-ui-materialui/src/input/RadioButtonGroupInput.spec.js

```diff
@@ -147,57 +158,46 @@ describe('<RadioButtonGroupInput />', () => {
         const { queryByText } = render(
             <RadioButtonGroupInput
                 {...defaultProps}
-                meta={{ helperText: 'Can I help you?' }}
+                helperText="Can I help you?"
             />
         );
         expect(queryByText('Can I help you?')).not.toBeNull();
     });
 
     describe('error message', () => {
         it('should not be displayed if field is pristine', () => {
-            const { container } = render(
```

Comment:
please keep this test

Reply:
I actually kept it but added an error because you can't test it isn't here if it is not defined right ? I removed the `should not be displayed if field has been touched but is valid` test though as I don't see its point